### ISSUE TITLE
Start torrent when added does not work

### ISSUE THREAD

**Issue by thesugarat** [transmission/transmission#4921]
### What is the issue?

When adding a torrent via magnet link through transmission remote. The torrent does not start automatically. However, when adding a torrent file through the web-ui it works correctly.

### Which application of Transmission?

transmission-remote

### Which version of Transmission?

4.0.0-r6-ls155

**Comment by dongennl** [comment]
same for me, it seems to work some of the time, but not always (when added through API from Sonarr)

**Comment by Coeur** [comment]
We made recent changed. Is you experience on a build which is before or after #4856 ? And before or after #4895 ?

**Comment by dongennl** [comment]
Hi, I am running on a Linux docker image:  linuxserver/transmission:latest, which seems to have refreshed itself on 16/2/2023 (it's automated). According to the build script on docker hub it is Transmission version 4.0.0-r6. 
I experienced a paused add this morning, which was added automatically from Sonarr; usually these start immediately, I had to start it manually as it had been there overnight.

**Comment by thesugarat** [comment]
> We made recent changes. Is you experience on a build which is before or after #4856 ? And before or after #4895 ?

I am running a docker container transmission/transmission:latest inside the default web-ui its About says 280ace12f8.

Also, watching the process, it appears to download the metadata first, then go into stop and wait for start.

Hope any of that helps.

**Comment by Coeur** [comment]
https://github.com/transmission/transmission/commit/280ace12f8a2b99fdf144011cd4f6fed8be876fe is before the fix of https://github.com/transmission/transmission/pull/4856, so your issue is a duplicate of https://github.com/transmission/transmission/issues/4740. Please try a newer build.

**Comment by thesugarat** [comment]
Thanks! Agree it's a duplicate of #4740. I'll have to wait on an updated image from the Linuxserver folks to update their base. I am actually using the linuxserver/transmission:latest
